core/src/rpc_pubsub.rs

```diff
@@ -24,6 +24,8 @@ use std::{
     sync::{atomic, Arc},
 };
 
+const MAX_ACTIVE_SUBSCRIPTIONS: usize = 100_000;
```

Comment:
100_000 is just a guess for a good max I gather?

Reply:
Yes, exactly, although we think tuning the `max_in_buffer_capacity` is probably more important, once that method is available in a new jsonrpc-pubsub version.